dashboards/TDinsight.json

```diff
@@ -6007,5 +6007,5 @@
   "timezone": "",
   "title": "TDinsight",
   "uid": "tdinsight",
-  "version": 28
+  "version": 31
```

Comment:
another comment: why skip two numbers?

Reply:
It's rev 30 in grafana website. Missing bump in last two versions.